dev/CommonStyles/RepeatButton_themeresources.xaml

```diff
@@ -84,11 +86,12 @@
 
     <Style x:Key="DefaultRepeatButtonStyle" TargetType="RepeatButton">
         <Setter Property="Background" Value="{ThemeResource RepeatButtonBackground}" />
-        <contract7Present:Setter Property="BackgroundSizing" Value="OuterBorderEdge" />
+        <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />
         <Setter Property="Foreground" Value="{ThemeResource RepeatButtonForeground}" />
         <Setter Property="BorderBrush" Value="{ThemeResource RepeatButtonBorderBrush}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ButtonBorderThemeThickness}" />
-        <Setter Property="Padding" Value="{StaticResource ButtonPadding}" />
+        <contract7NotPresent:Setter Property="Padding" Value="11,5,11,5" />
```

Comment:
>11,5,11,5 [](start = 62, length = 9)

Is there a reason for a hardcoded value here ?

Reply:
Not sure about that, in the Button style it was hardcoded
@savatia I believe you brought this padding, could you please tell what was the reason?

Reply:
@teaP  ?

Reply:
I think this 1 pixel is because of the inner border edge being different.